deployment/existing/auth_oidc/envoy-filter.yaml

```diff
@@ -19,6 +19,10 @@ spec:
           allowedHeaders:
             patterns:
             - exact: "cookie"
+        authorizationResponse:
+          allowedUpstreamHeaders:
+            patterns:
+            - exact: "kubeflow-userid"
```

Comment:
This will be a fix header and as I think it will give access to all requests with this header? 
But for userA who like to visit notebooks of userB, this filter will still authorize request?

Reply:
@Jeffwan the AuthService will add this header to every request.
If a malicious userA tries to specify a `kubeflow-userid` header to impersonate userB, the AuthService will prevent that, since it overrides the header with the correct value.